integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/OperatorUtils.java

```diff
@@ -280,6 +280,7 @@ public static OperatorParams installAndVerifyOperator(String opNamespace,
     OperatorParams opParams = new OperatorParams()
         .helmParams(opHelmParams)
         .imagePullSecrets(secretNameMap)
+        .javaLoggingLevel("FINE")
```

Comment:
Will this not cause a ton of information for all the tests. Can we create the OperatorParam on the tests with the FINE logging level.

Reply:
I agree... What is it that you're looking for that you would need the FINE level?

Reply:
changed code to use FINE logging only for one test

Reply:
Okay, but why would we commit this change? Why not use it only when trying to diagnose this problem?

Reply:
it is intermittent and happen only in OKE run, I have run OKE nightly against this branch for 2 days and can't get it failed anymore :).